gato/cli/cli.py

```diff
@@ -72,6 +72,18 @@ def cli(args):
         required=False,
     )
 
+    parser.add_argument(
+        "--api-url", "-u",
+        help=(
+            f"{Fore.RED}{bright('!! Experimental !!')}\n"
+            "Github API URL to target. \n"
+            "Defaults to 'https://api.github.com'"
+        ),
+        metavar="https://api.github-url.com",
```

Comment:
I think we need to update this to specify that you need to pass `https://HOSTNAME/api/v3/`

See: https://docs.github.com/en/enterprise-server@3.8/rest/actions/self-hosted-runners?apiVersion=2022-11-28#list-self-hosted-runners-for-an-organization